web/libs/datamanager/src/components/App/App.scss

```diff
@@ -7,15 +7,15 @@ body {
   --white: rgb(var(--white-raw));
   --accent_color: rgb(var(--accent_color-raw));
   --danger_color: rgb(var(--danger_color-raw));
-
-  --black_2:  rgba(var(--black-raw), 0.02);
```

Comment:
Catch this with @yyassi-heartex , decide if 

 --white-raw: 255, 255, 255; go to  --white-raw: 255 255 255;
 Or disable this rule.

Reply:
I will send a review commit for this soon :)

Reply:
changed